src/features/projectsV2/ProjectDetails/components/microComponents/ImageSlider.tsx

```diff
@@ -10,8 +10,14 @@ interface ImageSliderProps {
   images: Image[];
```

Comment:
Since we don't always pass in the images property from the projects API, it may be better to define the type for Image that is used in ImageSlider rather than refer to the common type defined. Makes the code easier to read.